.gitignore

```diff
@@ -9,6 +9,20 @@ __pycache__/
 # C extensions
```

Comment:
It's odd that we didn't start with something like
https://github.com/github/gitignore/blob/main/Python.gitignore
and then add on to it.

Reply:
I think we did and then all the merging and rebasing mucked it up. I'm trying to cleanup the duplicates in it.